engine/common/sys_con.c

```diff
@@ -230,6 +230,9 @@ void Sys_PrintLog( const char *pMsg )
 	time( &crt_time );
 	crt_tm = localtime( &crt_time );
 
+	// strip color codes
+	COM_StripColors( pMsg, (char *)pMsg );
```

Comment:
Here you can replace printf by function similar to Sys_PrintColorized that will print but ignoring color codes. Except try to avoid copying, it's useless, and I don't know why PrintColorized was made that way.